src/Symfony/Component/Serializer/Normalizer/AbstractNormalizer.php

```diff
@@ -283,7 +283,7 @@ protected function instantiateObject(array $data, $class, array &$context, \Refl
                 } elseif ($constructorParameter->isOptional()) {
                     $params[] = $constructorParameter->getDefaultValue();
                 } else {
-                    throw new RuntimeException(
+                    throw new \RuntimeException(
```

Comment:
it should use the RuntimeException of the component instead (and this case should be covered by tests btw)